touch-layouts/layouts/cyberpunk-2077.json

```diff
@@ -0,0 +1,371 @@
+{
+  "schema_version": 1,
+  "layouts": {
+    "LAYOUT_ID": {
```

Comment:
This should be changed to `custom` (same value as `default_layout` in other file)